flutter-for-ios.md

```diff
@@ -27,3 +27,429 @@ that are most relevant to your needs.
 
 * TOC Placeholder
 {:toc}
+
+# Views 
+
+## What is the equivalent of a `UIView` in Flutter?
+
+On iOS, the majority of anything you may want to create in the UI is done using
+view objects, which are instances of the `UIView` class.  These can act as
+containers for other `UIView` classes, which form your layout.
+
+In Flutter, the equivalent to this is a `Widget`. However, these have a few
+differences to a  `UIView`. To start, widgets only exist for one frame, and on
+every new frame, Flutter’s framework creates a new tree of widget instances.
+In comparison, an iOS view is drawn once and does not redraw until it is
+invalidated using `setNeedsDisplay()`.
+
+Furthermore, unlike `UIView`, Flutter’s widgets are immutable, which allows
+them to be very lightweight. By default, Flutter’s widgets are styled as
```

Comment:
Not by default, only if you select the MaterialApp and use the Material widgets. You can start with Cupertino if you want. Think about wording as we don't want to give the impression that we're iOS styling unfriendly.